uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/my-leave/apis/entitlement-api_test.js

```diff
@@ -0,0 +1,55 @@
+define([
+  'leave-absences/shared/modules/apis',
+  'leave-absences/shared/apis/entitlement-api',
+], function() {
+  'use strict';
+
+  describe('EntitlementAPI', function (){
+    var entitlementAPI;
+
+    beforeEach(module('leave-absences.apis'));
```

Comment:
the call is made to EntitlementAPI end points like say all but the URL is serviced by $httpBackend as described in mocks/apis/entitlement-api-mock.js. So instead of returning data from civicrm it will be returned by 
`
    instead of request going to this URL
    //civicrm/ajax/rest?action=get&entity=LeavePeriodEntitlement
    it will be served from here
    $httpBackend.whenGET(/action=get&entity=LeavePeriodEntitlement/)
      .respond(mockData.all_data);
`